<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wordcloud</class>
 <widget class="QMainWindow" name="Wordcloud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>435</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Topics</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>490</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Find Tweets on Topic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>10</y>
      <width>125</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Your Wordcloud</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>10</y>
      <width>125</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Twitter Wordcloud</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>340</width>
      <height>375</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>340</width>
      <height>375</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
